packages/examples/cvat/exchange-oracle/tests/api/test_exchange_api.py

```diff
@@ -38,7 +38,9 @@ def test_empty_list_tasks_200_without_address(client: TestClient) -> None:
 
 
 def test_list_tasks_200_with_address(client: TestClient) -> None:
-    with (SessionLocal.begin() as session,):
+    with (
+        SessionLocal.begin() as session,
+    ):
```

Comment:
```suggestion
    with SessionLocal.begin() as session:
```

Reply:
Fixed here and in other instance, thanks!